tests/integration/views/test_static.py

```diff
@@ -15,7 +15,7 @@ def test_contact(self, mock_contact):
         self.get('/contact-us')
         self.assertInPage('123456789012A')
 
-    def test_contact_with_no_metadata(self):
+    def test_contact_with_no_session(self):
         self.get('/contact-us')
         self.assertInPage('This office is closed during public holidays')
```

Comment:
You've just changed the name. This isn't checking the differences on the page when session_data is available compared to when it's not. This test would pass with or without session_data.